src/pages/AddServers.js

```diff
@@ -32,34 +34,90 @@ class AddServers extends BaseUpdateWizardPage {
     this.state = {
       model: this.props.model,
       deployedServers: undefined,
+      validating: false,
       // loading errors from wizard model or progress loading
-      loadingErrors: this.props.loadingErrors,
+      wizardLoadingErrors: this.props.wizardLoadingErrors,
+      // loading indicator from wizard
+      wizardLoading: this.props.wizardLoading,
       // errors from getting deployed servers
-      errors: undefined
+      // it is fatal so it shows as error banner across the page
+      errorBanner: undefined,
+      // error message show as a popup modal for validation errors
+      validationError: undefined,
+      // indicator of this loading
+      loading: true,
+      // show confirm dialog when user clicks Deploy
+      showDeployConfirmModal: false
     };
   }
 
   componentWillMount() {
-    fetchJson('/api/v1/clm/model/deployed_servers')
+    fetchJson('/api/v1/clm/model/deployed_servers', undefined, true, true)
```

Comment:
it might be nice to know what the undefined is standing in for, but not an urgent fix

Reply:
k  undefined is the init param for fetchJson...will add comments